### PR TITLE
Costume professions

### PR DESCRIPTION
**Pull request by Termineitor244**
#### Summary
Content "Adds or modifies professions that use costume clothing"

#### Purpose of change
Continuation of #56517, with some professions that start with sets of these items for more flavor and roleplay.

#### Describe the solution
Magiclysm:
Adds vampire clothing/jewelry to the Bionic Pseudovamp, some flavor items to the Novice Necromancer and a new profession: Grim Reaper! Also adds two new magic items: A sinister (blood spell) staff, and a real grim reaper scythe (with rupture spell).
Base game:
Adds Ninja, Cheerleader and Patriot professions to the base game.

#### Describe alternatives you've considered
To add enchanted items to the magicysm professions, like maybe a real underworld scythe, or a vampire ring that regenerates HP by sucking blood or something.

#### Testing
Tested the Vampire, Grim Reaper, Ninja, Cheerleader and Patriot professions and their new items, everything seems to work as intended.

#### Additional context
I accept suggestions for more costume clothed professions, I want to add shurikens and other yet to be introduced flavor items to the professions if I can, but that could wait for another PR, so do take that in mind.

These professions would really benefit of some kind soul adding custom worn graphics to them...

### CODE REVIEW

**Comment by AskaHope** [comment]
I think American Man/Woman is unnecessary. Considering the Cataclysm happened in New England, everyone there should be American by nature.

I think something like "Patriot" is more suitable.

**Comment by Termineitor244** [comment]
> I think American Man/Woman is unnecessary. Considering the Cataclysm happened in New England, everyone there should be American by nature.
> 
> I think something like "Patriot" is more suitable.

I was thinking of the name like a super hero name (Super-Man, American-Man) for someone obsessed with their country (To the point of using flag pants, t-shirt, underwear and been wrapped in the american flag), but yeah, I think patriot is better.

**Comment by Amoebka** [comment]
Is it possible for hobbies to override starting clothing or set up an EoC that will overwrite starting clothing? Making an entire profession just for clothing clutters the character creation.

**Comment by Termineitor244** [comment]
> Is it possible for hobbies to override starting clothing or set up an EoC that will overwrite starting clothing? Making an entire profession just for clothing clutters the character creation.

I'm not sure that hobbies can do that, but it would be cool, I will have to see. Though normally professions are mostly about initial clothing, with a few skill points here and there, but the majority of low leveled professions is just that, clothing,

File: data/json/professions.json [hunk 1]
```diff
@@ -4473,5 +4479,58 @@
...
+      "both": { "items": [ "pom_poms", "cheerleader_top", "socks_ankle", "sneakers" ] },
+      "male": [ "briefs", "shorts", "tank_top" ],
+      "female": [ "sports_bra", "panties", "cheerleader_skirt" ]
+    }
+  },
+  {
+    "type": "profession",
+    "id": "patriot",
+    "name": "Patriot",
+    "description": "You are a true american patriot, dressed with star-spangled clothing from head to toes.  The government has fallen, but the nation lives on with its people.",
```
Comment (TheShadowFerret) [inline data/json/professions.json:4525]:
```suggestion
    "description": "You are a true American patriot, dressed with star-spangled clothing from head to toes.  The government has fallen, but the nation lives on with its people.",
```

File: data/json/professions.json [hunk 2]
```diff
@@ -4473,5 +4479,58 @@
...
+      "traits": [ "LIGHTSTEP" ]
+    }
+  },
+  {
+    "type": "profession",
+    "id": "cheerleader",
+    "name": "Cheerleader",
+    "description": "Your last cheerleader routine ended abruptly after your partners started cheering for the unliving team.  Let's hope your quick reflexes and flexibility helps you to found new members for the squad.",
+    "points": 1,
+    "skills": [ { "level": 2, "name": "dodge" }, { "level": 2, "name": "swimming" } ],
```
Comment (TheShadowFerret) [inline data/json/professions.json:4514]: No social skill? 🤔

Comment (Termineitor244) [reply data/json/professions.json:4514]: You are right, good catch, I will add it now.

Comment (Termineitor244) [reply data/json/professions.json]: Done, I had to increase the point cost, so I increased by one point the athletics skill to compensate.

File: data/json/professions.json [hunk 3]
```diff
@@ -4473,5 +4479,58 @@
...
+      "both": {
+        "items": [ "hood_ninja", "jacket_ninja", "trousers_ninja", "tabi_dress", "geta", "gloves_wraps", "bellywrap" ],
+        "entries": [
+          { "item": "katana", "container-item": "scabbard" },
+          { "item": "grenadebandolier", "contents-group": "ninja_grenade_pouch" }
+        ]
+      },
+      "male": [ "boxer_shorts" ],
+      "female": [ "chestwrap", "panties" ],
+      "traits": [ "LIGHTSTEP" ]
```
Comment (dseguin) [inline data/json/professions.json:4505]:
```suggestion
    ],
     "traits": [ "LIGHTSTEP" ],
    "items": {
      "both": {
        "items": [ "hood_ninja", "jacket_ninja", "trousers_ninja", "tabi_dress", "geta", "gloves_wraps", "bellywrap" ],
        "entries": [
          { "item": "katana", "container-item": "scabbard" },
          { "item": "grenadebandolier", "contents-group": "ninja_grenade_pouch" }
        ]
      },
      "male": [ "boxer_shorts" ],
      "female": [ "chestwrap", "panties" ]
```

Comment (Termineitor244) [reply data/json/professions.json]: Agh, I had already fixed (forgetting commas...) in my local branch, but thanks!

Comment (dseguin) [reply data/json/professions.json]: Nevermind, I see that you fixed it just now :P

**Comment by Termineitor244** [comment]
All tests green, but I'm not sure if I should wait for a review of KorGgenT for the Magiclysm changes...

And speaking of Magiclysm!... Making this PR has left me wanting a professions flavor/balance update for Magiclysm, like the ones I made for the base game and the bionic professions mod some time ago... That's the reason why I keep myself away from these files, I can not contain myself from adding flavor (useless) items to the professions I see...

**Comment by Termineitor244** [comment]
Argh, I created conflicts for myself haha. Well, at least they were easy to resolve.

**Pull request merged**